components/src/core/drawer/drawer-header/drawer-header.component.scss

```diff
@@ -0,0 +1,53 @@
+.pxb-drawer-header {
+    padding-right: 0;
+    padding-left: 0;
+    width: 100%;
+    align-items: flex-start;
+    box-sizing: border-box;
+}
+
+.pxb-drawer-header-icon-wrapper {
+    padding: 4px;
+    height: 64px;
+    display: flex;
+    align-items: center;
+    z-index: 1;
+}
+
+.pxb-drawer-header-wrapper {
```

Comment:
shouldn't this be `pxb-drawer-header-title-wrapper`? otherwise drawer-header-wrapper is below drawer-header in the tree